modules/common-jpa-impl/src/main/java/org/opencastproject/security/impl/jpa/JpaGroup.java

```diff
@@ -81,13 +81,18 @@
   private String role;
 
   @ElementCollection
-  @CollectionTable(name = "oc_group_member", joinColumns = { @JoinColumn(name = "group_id") })
+  @CollectionTable(name = "oc_group_member", joinColumns = {
+      @JoinColumn(name = "group_id", nullable = false) })
```

Comment:
This generates a foreign key constraint to `oc_group` which seems to be correct but is not in the db script